Description: Do not check socket path truncation on hurd either
 On GNU/Hurd perl's $sock->hostpath() can't work since the path is not recorded
 in the socket internals. GNU/Hurd doesn't have any socket path limitation
 anyway, so the attached proposed patch just skips the check.
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>

--- a/mysql-test/lib/My/Platform.pm.original	2016-10-31 00:31:33.000000000 +0000
+++ b/mysql-test/lib/My/Platform.pm	2016-10-31 00:32:37.000000000 +0000
@@ -112,6 +112,8 @@
   return 0 if ($^O eq 'aix');
   # See Debian bug #670722 - failing on kFreeBSD even after setting short path
   return 0 if $^O eq 'gnukfreebsd' and length $path < 40;
+  # GNU/Hurd doesn't have hostpath(), but no limitation either.
+  return 0 if $^O eq 'gnu';
 
   require IO::Socket::UNIX;
 
